variants/fcos/okd-rpms/manifests/99-okd-master-disable-mitigations.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: machineconfiguration.openshift.io/v1
```

Comment:
Ditto, we should build SCOS with these options enabled (not sure if intel_pstate is necessary anymore)

Reply:
@vrutkovs we don't build scos here yet as we consume it directly from the artifacts produced by the okd-project/okd-scos pipeline.

Ideally, we should reach a target like:

- okd-project/okd-scos produces a base scos image (w/o okd-specific content)
- we build the layered okd-specific scos as we do for fcos (leveraging a third repo similar to openshift/okd-machine-os)
- 
right?